src/plone/restapi/tests/http-examples/navroot_site_get.req

```diff
@@ -0,0 +1,2 @@
+GET /plone/@navroot HTTP/1.1
```

Comment:
I suggest an example for an arbitrary page, not the navroot itself.

Reply:
done that too.